script/make-thci.bash

```diff
@@ -33,14 +33,32 @@ echo "OUTPUT_ROOT=${OUTPUT_ROOT?}"
 
 mkdir -p "$OUTPUT_ROOT"/ot-comm
 
+# Args:
+# - $1 - src_path: Source path of the THCI file
+# - $2 - out_name: Name of the output file
+ncs_adapt()
+{
+  echo ${1}
+  local src_path=${1}
+  local out_path="$OUTPUT_ROOT"/${2}
+
+  cp ${src_path} ${out_path}
+  sed -i 's/Device : OpenThread/Device : OTNCS/g' ${out_path}
+  sed -i 's/Class : OpenThread/Class : OTNCS/g' ${out_path}
+  sed -i 's/class OpenThread/class OTNCS/g' ${out_path}
+  sed -i 's/THCI.OpenThread/THCI.OTNCS/g' ${out_path}
+}
+
+src_dir=openthread/repo/tools/harness-thci
```

Comment:
shouldn't it be openthread/tools/harness-thci? (without /repo/)

Reply:
Yes, thanks. Changed.